src/main/haskell/language-kore/src/Data/Kore/Variables/Free.hs

```diff
@@ -17,14 +20,18 @@ provides 'freeVariables' for extracting the set of free variables of a term
 class TermWithVariablesClass term var | term -> var where
     freeVariables :: term -> Set.Set var
 
-instance VariableClass var
-    => TermWithVariablesClass (FixedPattern var) (UnifiedVariable var) where
+instance
+    ( Typeable var
+    , Ord (var Object)
```

Comment:
This `Ord (var Object), Ord (var Meta)` pattern keeps repeating. Is there any way of saying something like `Ord (ObjectOrMeta var)`?

Reply:
Not like that, but you can make type synonyms for parts of class constraints with the `ConstraintKinds` extension.
Something like `type ObjectOrMeta1 cls var = (cls (var Object), cls (var Meta))`